Flash Chat iOS13/AppDelegate.swift

```diff
@@ -12,36 +12,50 @@ import Firebase
-
-	var userEmails: [String] = []
-
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+	
+	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+		// Override point for customization after application launch.
 		configureKeyboard()
 		FirebaseApp.configure()
-        return true
-    }
-
+		return true
+	}
+	
 	func configureKeyboard() {
 		IQKeyboardManager.shared.enable = true
 		IQKeyboardManager.shared.enableAutoToolbar = false
 		IQKeyboardManager.shared.shouldResignOnTouchOutside = true
 	}
-
-    // MARK: UISceneSession Lifecycle
-
-    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
-        // Called when a new scene session is being created.
-        // Use this method to select a configuration to create the new scene with.
-        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
-    }
-
-    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
-        // Called when the user discards a scene session.
-        // If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
-        // Use this method to release any resources that were specific to the discarded scenes, as they will not return.
-    }
-
+	
+	// MARK: UISceneSession Lifecycle
+	
+	func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
+		// Called when a new scene session is being created.
+		// Use this method to select a configuration to create the new scene with.
+		return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
+	}
+	
+	func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
+		// Called when the user discards a scene session.
+		// If any sessions were discarded while the application was not running, this will be called shortly after application:didFinishLaunchingWithOptions.
+		// Use this method to release any resources that were specific to the discarded scenes, as they will not return.
+	}
+	
+	static func checkRecipientRegistrationStatus(_ recipient: String, inDatabase database: Firestore, completionHandler: @escaping ((Bool, Error?) -> Void)) {
+		database.collection(Constants.FStore.usersCollectionName)
+			.whereField(Constants.FStore.emailField, isEqualTo: recipient)
+			.getDocuments { users, error in
+				if let error = error {
```

Comment:
Creating the function `checkRecipientRegistrationStatus` is an excellent refactor!

In the future, please try to place such a refactor into its own commit; in this case, it's contained in a commit named `Fixed Indentation`, which can make it a bit more challenging for the reviewer, if they're expecting only indentation changes in that commit.

Reply:
The function was implemented in the previous commits, not this indentation one.